ast/ast_test.go

```diff
@@ -10,10 +10,8 @@ import (
 	"github.com/cedar-policy/cedar-go/types"
 )
 
-// These tests serve mostly as examples of how to translate from Cedar text into programmatic AST construction. They
-// don't verify anything.
-func TestAstExamples(t *testing.T) {
-	t.Parallel()
+// This example shows how you can construct polcies with the ast package
+func ExampleNode() {
```

Comment:
Yes, thank you! We've been meaning to do this.

I think we could actually elevate these up to just `Example()` so that they appear right at the top of the documentation.